### DISCUSSION TITLE
Include the Provider Signer in the repository.

### DISCUSSION THREAD

**Discussion by niACS** [ni/actor-framework#59] [General]
**Is your feature request related to a problem? Please describe.**
Developers working on project providers need to digitally sign their provider's INI file.  NI has a VI that does so.

**Describe the solution you'd like**
Include this tool in the repository.

**Describe alternatives you've considered**
There are none.  Developers need the tool.

**Additional context**
The tool is unofficially available on the LabVIEW WIKI, but developers shouldn't have to hunt for it.  I'd also like to know that I'm using the actual tool from NI.

**Comment by gregr-ni** [comment]
The provider signer should have its own home, not be copied into any individual client repo. The task here would be to consume that export as a dependency. Thoughts on what that official export should be would be welcome.

**Comment by svelderrainruiz** [comment]
> The provider signer should have its own home, not be copied into any individual client repo. The task here would be to consume that export as a dependency. Thoughts on what that official export should be would be welcome.

@gregr-ni i can own creating a repo for the provider signer, and creating a pipeline that generates the package. Allen can then use this dependency on its package.

**Comment by niACS** [comment]
Sounds good to me.

Allen C Smith, CLA

**Comment by svelderrainruiz** [comment]
ok,  @gregr-ni will provide the official version to me and i will create a repo + vi package release.
